processing/src/main/java/io/druid/segment/Metadata.java

```diff
@@ -250,12 +248,7 @@ public boolean equals(Object o)
   @Override
   public int hashCode()
   {
-    int result = container.hashCode();
-    result = 31 * result + Arrays.hashCode(aggregators);
-    result = 31 * result + (timestampSpec != null ? timestampSpec.hashCode() : 0);
-    result = 31 * result + (queryGranularity != null ? queryGranularity.hashCode() : 0);
-    result = 31 * result + (rollup != null ? rollup.hashCode() : 0);
-    return result;
+    return Objects.hash(container, Arrays.hashCode(aggregators), timestampSpec, queryGranularity, rollup);
```

Comment:
`equals` could be simplified too (using Objects.equals and Arrays.equals)

Reply:
Done.